source/includes/cxi/_segments.md

```diff
@@ -0,0 +1,43 @@
+# Segments/Properties
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+resposes[][id] | integer | Response ID
```

Comment:
`response` 😬

Reply:
Actually, the endpoint is expecting [responses](https://github.com/Wootric/wootric-ai-app-backend/blob/master/app/controllers/v1/segments_controller.rb#L51)

Reply:
Then it should be `responses`, it's missing the `n`

Reply:
![facepalm](https://user-images.githubusercontent.com/747158/124803026-f8c3c000-df2e-11eb-8148-08ee7d439f02.gif)